website/docs/r/event_orchestration_resource.html.markdown

```diff
@@ -28,166 +23,243 @@ resource "pagerduty_event_orchestration" "my_monitor" {
   team {
-    id = pagerduty_team.foo.id
+    id = pagerduty_team.engineering.id
   }
 }
 ```
 
-## Example of configuring Router rules for an Orchestration
+## Argument Reference
 
-```hcl
-resource "pagerduty_event_orchestration_router" "my_monitor" {
-  event_orchestration = pagerduty_event_orchestration.my_monitor
-  # Note: the router currently only supports a single set of rules.
-  # If we change routers to support multiple sets in the future we'd need to introde a `sets` argument.
-  rules = [
-    {
-      name = "Events relating to our relational database"
-      conditions = [
-        { expression = "event.summary matches part 'database'" },
-        { expression = "event.source matches regex 'db[0-9]+-server'" }
-      ]
-      route_to = pagerduty_service.database.id
-    },
-    {
-      name = "Events relating to our www app server"
-      conditions = [
-        { expression = "event.summary matches part 'www'" }
-      ]
-      route_to = pagerduty_service.www.id
-    }
-  ]
-  catch_all = {
-    route_to = "unrouted"
-  }
-}
-```
+The following arguments are supported:
 
-**Argument Reference:** supports same rule and catch-all arguments as the [Router Orchestration endpoint](https://developer.pagerduty.com/api-reference/b3A6MzU3MDU0Mzk-update-the-router-for-a-global-event-orchestration) of the Public API
+* `name` - (Required) Name of the Event Orchestration.
+* `description` - (Optional) A human-friendly description of the Event Orchestration.
+* `team` - (Optional) Reference to the team that owns the Event Orchestration. If none is specified, only admins have access.
 
-## Example of configuring a Unrouted Rules for an Orchestration
+## Attributes Reference
 
-```hcl
-resource "pagerduty_event_orchestration_unrouted" "my_monitor" {
-  event_orchestration = pagerduty_event_orchestration.my_monitor
-  sets = [
-    {
-      id = "start"
-      rules = [
-        {
-          name = "Update the summary of un-matched Critical alerts so they're easier to spot"
-          conditions = [
-            { expression = "event.severity matches 'critical'" }
-          ]
-          actions {
-            extractions = [
-              {
-                target = "event.summary"
-                template = "[Critical Unrouted] {{event.summary}}"
-              }
-            ]
-          }
-        },
-        {
-          name = "Reduce the severity of all other unrouted events"
-          conditions = []
-          actions = {
-            severity = "info"
-          }
-        }
-      ]
-    }
-  ]
-  # In this example the user has defined their own "all other unrouted events" rule with no conditions
-  # so they aren't defining global catch_all behavior for the pagerduty_event_orchestration_unrouted resource
-  # but using the catch_all would be a totally legit alternative.
-}
+The following attributes are exported:
+
+* `id` - The ID of the Event Orchestration.
+* `integrations` - Routing keys routed to this Event Orchestration.
```

Comment:
**Question:** Won't `integrations` be an Array of objects? I wonder if we should somehow document the attributes available for each `integration`.

**Bonus:** Demonstrate how use the attribute to integrate with an event-provider like DataDog: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/integration_pagerduty_service_object